docs/components_page/components/input_group/dropdown.py

```diff
@@ -22,21 +23,22 @@
 @app.callback(
     Output("input-group-dropdown-input", "value"),
     [
-        Input("dropdown-menu-item-1", "n_clicks_timestamp"),
-        Input("dropdown-menu-item-2", "n_clicks_timestamp"),
-        Input("dropdown-menu-item-clear", "n_clicks_timestamp"),
+        Input("dropdown-menu-item-1", "n_clicks"),
+        Input("dropdown-menu-item-2", "n_clicks"),
+        Input("dropdown-menu-item-clear", "n_clicks"),
     ],
 )
 def on_button_click(n_clicks_t1, n_clicks_t2, n_clicks_tclear):
```

Comment:
Does it make sense to change the argument names? Most commonly, unused arguments are replaced by a `_`, but obviously that won't work for three buttons.

Reply:
At least removing the `t` in the argument name since we're catching `n_clicks`, not `n_clicks_timestamp`.

Reply:
Good point, I'll amend